docs/GettingStarted.md

```diff
@@ -288,6 +288,15 @@ current_span = Datadog::Tracing.active_span
 current_span.set_tag('my_tag', 'my_value') unless current_span.nil?
 ```
 
+You can record an exception in the current span. It will add the recorded exception as a span event. If you also want to set the span error tags, you should set `escaped` to true.
```

Comment:
The example code is broken due to 

1. It is invoking with positional argument, not keyword arguments
2. Does not handle correctly when `Datadog::Tracing.active_span` returns `nil`
3. `false` instead of `False` (why displaying a method default which could be ignored?)

Reply:
Yes, it is an opentelemetry term. It is implemented in Python for unhandled exceptions.
TL;DR: it is not in ruby so I'll just remove it